fixtures/nanlite/forza-300b.json

```diff
@@ -0,0 +1,262 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Forza 300B",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2023-06-27",
+    "lastModifyDate": "2023-06-27"
+  },
+  "links": {
+    "manual": [
+      "https://download.s21i.faiusr.com/22292231/0/0/ABUIABA9GAAg_IGthgYokNjW-QY.pdf?f=TK-140B%EF%BC%8FTK-280B+Manual.pdf&v=1623933177",
+      "https://download.s21i.faiusr.com/22292231/0/0/ABUIABA9GAAg_YGthgYokLmTxgQ.pdf?f=TK-140B%EF%BC%8FTK-280B+DMX+REFERENCE+GUIDE.pdf&v=1623933177"
```

Comment:
This says "TK-140B/TK-280B DMX REFERENCE GUIDE", which looks like a very different fixture than the Forza 300B.

Reply:
Fixed in [`3ea4b85`](https://github.com/OpenLightingProject/open-fixture-library/pull/3232/commits/3ea4b851a362684937bc554b1d30c6137563fb02).